docs/development/release.md

```diff
@@ -72,81 +132,99 @@ Similar to #4, make sure the DATA_MODEL_CHANGELOG is updated if there are data m
 git add CONTRIBUTORS.md
 ```
 
+Commit and push these changes, if any.
+
 ### 6. Update Cumulus package API documentation
 
 Update auto-generated API documentation for any Cumulus packages that have it:
 
 ```bash
-  npm run docs-build-packages
+npm run docs-build-packages
 ```
 
+Commit and push these changes, if any.
+
 ### 7. Cut new version of Cumulus Documentation
 
-If this is a backport, do not version documentation. For various reasons, we do not merge backports back to master, other than changelog notes. Doc changes will not be published to our documentation website.
+> Note: This is for version `1.10.3` or later.
```

Comment:
Is the 1.10.3 comment *useful* at this point?        I can't imagine needing to cut a release < 1.10.3.